1/index.js

```diff
@@ -1 +1,11 @@
-console.log('1!');
+//lesson01--------------------------------------------------
+//chatGPT prompt："javascript DOM作成 li テキスト「これです」"
+//reference:https://www.wakuwakubank.com/posts/306-javascript-dom/#index_id4
+//reference:https://stackoverflow.com/questions/50250214/how-to-use-insertbefore-without-second-parameter
+
+//create list item
+const newList = document.getElementById('newlist'); // get:ID='newlist'
+console.log(newList);
+const listItem = document.createElement('li'); // create<li></li>
+const newContent =  listItem.textContent = 'これです' // add:text -> これです
+newList.insertBefore(listItem , null); 
```

Comment:
IMO
insertBeforeの第二引数に明示的にnullを指定する必要もなくなるので[appendChild](https://developer.mozilla.org/ja/docs/Web/API/Node/appendChild)を使用するでもいいと思いました！

```suggestion
newList..appendChild(listItem); 
```

Reply:
That's right!
That's more easy to write and understand!
Thank you!

Reply:
https://github.com/megusasuke0/frontendhandson/commit/d1cd14bfa81aaa8d39ce0a8fd4d90cb7be53f627